docs/release-notes/7.16.asciidoc

```diff
@@ -1,6 +1,6 @@
-[discrete]
+
```

Comment:
I think if you want this content to be merged into the "Release notes" page, you need to keep the discrete attribute:
```suggestion
[discrete]
```